### DISCUSSION TITLE
Ghostty crashes when using a hyperlink

### DISCUSSION THREAD

**Discussion by jan-rex** [ghostty-org/ghostty#4103] [Feature Requests, Ideas] [answered]
When using a hyperlink (pressing the Cmd key + clicking on the link), Ghosstty (716848cb) crashes abruptly

**Comment by 00-kat** [comment, marked as answer]
Two months later, does this still happen?

**Reply by jan-rex** [reply]
“With the updates from the following day, it was resolved.”
